drivers/counter/Kconfig.nrfx

```
@@ -10,7 +10,7 @@ config COUNTER_NRF_RTC
 config COUNTER_TIMER0
 	bool "Enable Counter on TIMER0"
 	depends on HAS_HW_NRF_TIMER0
-	depends on !BT_LL_SW_SPLIT
+	depends on !BT_CTLR
```

```suggestion
	depends on !BT_CTLR && !NRF_802154_MULTIPROTOCOL_SUPPORT
```
As @rugeGerritsen pointed out, `COUNTER_TIMER0` should not be used when 802.15.4 is in multiprotocol configuration